packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -1478,4 +1503,41 @@ void main() {
     await tester.pump();
     expect(textField.cursorColor, errorColor);
   });
+
+  testWidgetsWithLeakTracking('TextFormField onChanged is called when the form is reset', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/123009.
+    final GlobalKey<FormFieldState<String>> stateKey = GlobalKey<FormFieldState<String>>();
+    final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+    String value='initialValue';
```

Comment:
I think this can be const. And add a space between `value=`.

Reply:
const is not possible here because the variable is modified in line1521.
Good catch for the missing spaces, thanks!